sass/_a4-home-bottom.scss

```diff
@@ -0,0 +1,39 @@
+//home bottom styling
+//this includes some amazing features and icons below that
+.a4-home-bottom {
+  @extend %section-layouts;
+  background-color: $low-light;
+  & .slant {
+    @extend %svg;
+    fill: $background-C;
+    background-color: transparent;
+  }
+}
+.home-feature {
+	margin-bottom: 6em;
+}
+.home-feature-h5 {
+  color: $primary-font-color;
+}
+.home-feature figcaption {
+	font-size: $tiny-font-size;
+	max-width: 20em;
+	margin: 0 auto;
+  color: $primary-font-color;
+}
+.home-feature-laptop {
+	margin-top: 6em;
+}
+
+@media (min-width: 1024px) {
+  .home-feature-laptop {
+    margin-top: 1em;
+  }
+    .home-feature-list{
+      display: flex;
+      margin-left: 17%;
```

Comment:
I'm curious why you set this up this way -- the `margin-left` value here is pushing your content a bit out of alignment and as a result it's off-center. A better solution might have been to use a variable width and margin-auto:

```scss
.home-feature-list {
  display: flex;
  width: 80%; // or whatever makes the most sense
  margin: { // if you don't want to set the top/bottom to 0, other wise you can just shorthand it to `margin: 0 auto;`
    right: auto;
    left: auto;
  }
  text-align: center;
}
```